GameOfLife/Calculations/RandomCellGenerator.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace GameOfLife
+{
+
+    public class RandomCellGenerator
+    {
+        // Generates randomly which cells are alive which not
+        public int[,] RandCells(int[,] GameField)
+        {
+            Random rand = new Random();
```

Comment:
Same for variable names, try not to shorten them, we got big monitors now, we can afford to read longer code lines :D